go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/ipfs/go-cid v0.0.3
 	github.com/ipfs/go-datastore v0.1.0
 	github.com/ipfs/go-ipfs-util v0.0.1
-	github.com/ipfs/go-log v0.0.1
+	github.com/ipfs/go-log v0.0.2-0.20190920042044-a609c1ae5144
```

Comment:
Let's update to https://github.com/ipfs/go-log/releases/tag/v2.0.2.

Reply:
@raulk yeah, I need to do this for all the go-log upgrade related PRs. Just been reprioritized for a day.